src/eventDispatcher.js

```diff
@@ -0,0 +1,19 @@
+function EventDispatcher () {
+	this.eventDispatcherMap = {};
+};
+
+EventDispatcher.prototype.bind = function(nameOfFunction, whichFunction) {
```

Comment:
Event dispatcher dispatches events, not functions:
`nameOfFunction` -> `eventName`
`whichFunction` -> `function` or 'handler`